px-tooltip.html

```diff
@@ -278,55 +316,78 @@
 
         //detached triggered by our show/hide system
         this._internalDetaching = false;
-        if(!this._requestShow) {
+        if(!this.requestShow) {
           this.fire('px-tooltip-hide', {target: this});
         }
       }
     },
 
-    _listenEvents: function() {
-      if(this._target) {
+    /**
+    * Starts listening to all mouse events (on the target(s)) we are
+    * interested in: mouseenter, mouseup, mouseleave, mousedown
+    */
+    listenEvents: function() {
```

Comment:
I have a concern now that these are explicitly public (and so there's no reason the developer doesn't use them): What if we add listeners more than once to a target? We don't check whether or not we've added listeners before, we just check if there is a target or targets, and that ignoreTargetEvents is false.

This could lead to the external dev calling `listenEvents` and the internal code calling `listenEvents` and listeners getting bound more than once, right? Unless `this.listen()` from Polymer is smart enough to not add multiple event listeners for the same event + function combo.

Reply:
Good points, I had to go double check, seems like this.listen is clever enough to not attach it several times even when called several time 👍